Description: Omit the resource data files from Python packaging config.
 .
 Created to avoid Python's Distutils insistence on install of
 arch-independent ‘package_data’ files in ‘/usr/lib/…’. This is a violation
 of FHS, and these files should instead go to ‘/usr/share/…’.
Bug-Debian: http://bugs.debian.org/721676
Author: Ben Finney <bignose@debian.org>
Last-Update: 2032-07-23

diff --git old/setup.py new/setup.py
--- old/setup.py
+++ new/setup.py
@@ -87,7 +87,6 @@ setup_args = dict(
 
     package_data={
         'coverage': [
-            'htmlfiles/*.*',
             'fullcoverage/*.*',
             'py.typed',
         ]

Local variables:
coding: utf-8
mode: diff
time-stamp-format: "%:y-%02m-%02d"
time-stamp-start: "^Last-Update:[ 	]+"
time-stamp-end: "$"
time-stamp-line-limit: 20
End:
vim: fileencoding=utf-8 filetype=diff :
